README.md

```diff
@@ -580,7 +580,7 @@ directly.
 │                                            │                    │                                              │
 │                                            │   ┌─────────────┐  │                                              │
 │                                            │   │             │  │                                              │
-│                   ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─│─ ▶│etcd cluster │◀ ┼ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─                    │
+│                   ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─│─ │etcd cluster │ ┼ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─                    │
```

Comment:
Hmm weird I didn't touch them at all. Only guess is my editor to offence to them somehow O.o